src/views/Disease/Disease.jsx

```diff
@@ -270,29 +279,32 @@ const Disease = ({ routeParamMap, routePath, data }) => {
 		<>
 			{renderHelmet()}
 			<h1>{replacedText.pageTitle}</h1>
-			{replacedText.introText.length > 0 &&
-				trialsPayload?.trials.length > 0 && (
-					<div
-						className="intro-text"
-						dangerouslySetInnerHTML={{ __html: replacedText.introText }}></div>
-				)}
+			{replacedText.introText.length > 0 && fetchedAndHasTrials(fetchState) && (
```

Comment:
This code is crazy, @arcepaul and I discussed it and we really should not have so many things "rendering" above and below the spinner if they do not show during load.